docs/src/install.md

```diff
@@ -0,0 +1,7 @@
+# Build freenet from source On Linux:
+
+There is a single line command to build all freenet on Linux.
```

Comment:
I think this is a good idea, but I think it would work better as a script we could include in the root directory - perhaps called `build-all.sh`.

Doing it as a script would allow it to be clearer, and also to avoid steps like installing Rust if it is already installed.

chatgpt is quite good at this type of thing if you want to try it.

Reply:
done